third_party/libpng/tnt/CMakeLists.txt

```diff
@@ -51,5 +51,7 @@ add_library(${TARGET} STATIC ${PRIVATE_HDRS} ${PUBLIC_HDRS} ${SRCS})
 
 target_link_libraries(${TARGET} PUBLIC z)
 
+target_compile_definitions(${TARGET} PRIVATE PNG_ARM_NEON_OPT=0)
```

Comment:
This should only be disabled when compiling for Linux aarch64, we may need this when running on mobile devices.

Reply:
Done. Now we check the `LINUX_AARCH64` flag first.